### PR TITLE
The API for uploading files using the knowledge base name has been ad…

### PR DESCRIPTION
**Pull request by zhangyulei000**
…ded.

### Resolve the issue #417 . 

The API for uploading files using the knowledge base name has been added.

### Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

### CODE REVIEW

File: api/db/services/common_service.py [hunk 1]
```diff
@@ -118,6 +118,15 @@ def get_by_id(cls, pid):
         except Exception as e:
             return False, None
 
+    @classmethod
```
Comment (KevinHuSh) [inline api/db/services/common_service.py]: You could add this chunk of code to the knowledge base  service.

File: api/apps/api_app.py [hunk 2]
```diff
@@ -192,4 +200,75 @@ def get(conversation_id):
 
         return get_json_result(data=conv.to_dict())
     except Exception as e:
-        return server_error_response(e)
\ No newline at end of file
+        return server_error_response(e)
+
+
+@manager.route('/conversation/upload', methods=['POST'])
```
Comment (KevinHuSh) [inline api/apps/api_app.py]: hange this to /document/upload

File: api/apps/api_app.py [hunk 3]
```diff
@@ -192,4 +200,75 @@ def get(conversation_id):
...
         return get_json_result(data=conv.to_dict())
     except Exception as e:
-        return server_error_response(e)
\ No newline at end of file
+        return server_error_response(e)
+
+
+@manager.route('/conversation/upload', methods=['POST'])
+@login_required
+@validate_request("name")
```
Comment (KevinHuSh) [inline api/apps/api_app.py]: name->kb_name

**Comment by zhangyulei000** [comment]
Based on the review feedback, I have made the necessary fixes.

**Pull request closed without merging**